README.md

```diff
@@ -8,25 +8,39 @@
-  with `:GoTest` (run single tests via `:GoTestFunc`)
-* Show test coverage with `:GoCoverage` or in browser with `:GoCoverageBrowser` 
-* Goto definition with `:GoDef`
-* Quick jump to declarations with `:GoDecls` or `:GoDeclsDir`
-* Show documentation with `:GoDoc` inside or in browser with `:GoDocBrowser`
-* Quickly execute your current file/files with `:GoRun`
-* Advanced source analysis tools utilizing guru, such as `:GoImplements`,
-  `:GoCallees`, and `:GoReferrers`
-* Change or display `GOPATH` with `:GoPath`
-* Multiple 3rd linter support with `:GoMetaLinter`
-* Renaming identifiers with `:GoRename`
-* Share your code to [play.golang.org](http://play.golang.org) with `:GoPlay`
-* Switch between `*.go` and `*_test.go` code with `:GoAlternate`
-* Add/Remove tags on struct fields with `:GoAddTags`
-* Add import paths via `:GoImport` or remove them with `:GoDrop`
-* Custom vim text objects such as `a function (af)` or `inner function (if)`
-* ... and many more! Please see [doc/vim-go.txt](doc/vim-go.txt) for more information.
-
+* Improved syntax highlighting and folding.
+* Completion support via `gocode`.
+* `gofmt` or `goimports` on save keeps the cursor position and undo history.
+* Go to symbol/declaration with `:GoDef`.
+* Look up documentation with `:GoDoc` or `:GoDocBrowser`.
+* Easily import packages via `:GoImport`, remove them via `:GoDrop`.
+* Compile your package with `:GoBuild`, install it with `:GoInstall` or test it
+  with `:GoTest`. Run a single tests with `:GoTestFunc`).
+* Quickly execute your current file(s) with `:GoRun`.
+* Automatic `GOPATH` detection which works with `gb` and `godep`. Change or
+  display `GOPATH` with `:GoPath`.
+* See which code is covered by tests with `:GoCoverage`.
+* Add or remove tags on struct fields with `:GoAddTags` and `:GoRemoveTags`.
+* Call `gometalinter` with `:GoMetaLinter` to invoke all possible linters
+  (`golint`, `vet`, `errcheck`, `deadcode`, etc.) and put the result in the
+  quickfix or location list.
+* Lint your code with `:GoLint`, run your code through `:GoVet` to catch static
+  errors, or make sure errors are checked with `:GoErrCheck`.
+* Advanced source analysis tools utilizing `guru`, such as `:GoImplements`,
+  `:GoCallees`, and `:GoReferrers`.
+* Precise type-safe renaming of identifiers with `:GoRename`.
+* Integrated and improved snippets, supporting `ultisnips` or `neosnippet`.
+* Share your current code to play.golang.org with `:GoPlay`.
+* On-the-fly information about the word under the cursor. Plug it into your
+  custom Vim function.
+* Text objects such as "a function" (`af`) or "inner function" (`if`).
+* Most commands are run asynchronous in Neovim and Vim 8. Fully async building
+  and testing.
+* Integrated with the Neovim terminal, launch `:GoRun` and other Go commands in
+  a terminal buffer.
+* Switch between `file.go` and `file_test.go` code with `:GoAlternate`.
+* Supports integration with the Tagbar and ctrlp.vim plugins.
+* ...and more...
```

Comment:
Previously it was like this, but with ever growing features this was not maintainable. Therefore I've cut it down to show the most notable and best feature (Which are opinionated of course). For example, the debugging feature would be  added here, but things like `automated gopath detection` should be list here. 

I try to keep the README simple and minimalist. Would be happy if you remove the changes to README.md and we evaluate it later what to show and not.